usr/src/uts/common/fs/zfs/vdev.c

```diff
@@ -1431,6 +1433,7 @@ vdev_open(vdev_t *vd)
 		vd->vdev_asize = asize;
 		vd->vdev_max_asize = max_asize;
 		vd->vdev_ashift = MAX(ashift, vd->vdev_ashift);
+		vd->vdev_ashift = MAX(zfs_ashift_override, vd->vdev_ashift);
```

Comment:
it seems strange than an override really isn't an override, it is MAX() of the override and something else

Reply:
Does it seem strange to you because the default also serves as the minimum(and you'd like a separate minimum field) or is it using the term override when it is subject to restrictions (and you prefer something more like zfs_ashift_hint)?

Reply:
A better name is `zfs_ashift_min` which is a fine approach.

For the name `zfs_ashift_override` the expected code is something like:
`vd->vdev_ashift = zfs_ashift_override > 0 ? zfs_ashift_override : vd->vdev_ashift;`
which is also a fine approach.

Basically, I'd like to see it become less ambiguous and more consistent with other tunables.